src/fty_alert_stats.cc

```diff
@@ -64,8 +64,8 @@ int main (int argc, char *argv [])
 {
     const char * CONFIGFILE = "";
     const char * LOGCONFIGFILE = "";
-    const char * metricTTL = "";
-    const char * tickPeriod = "";
+    const char * metricTTL = "720";
```

Comment:
I see these magic numbers in math below in fty_alert_stats_server.cc as well. Should the numbers here or down there become variables or macros (or int/string conversions), to avoid discrepancies later on?

Reply:
Never mind, that is a `..._test()` there.